contracts/src/v0.8/functions/dev/1_0_0/FunctionsCoordinator.sol

```diff
@@ -158,7 +158,8 @@ contract FunctionsCoordinator is OCR2Base, IFunctionsCoordinator, FunctionsBilli
       subscriptionId: request.subscriptionId,
       client: request.requestingContract,
       callbackGasLimit: request.callbackGasLimit,
-      expectedGasPrice: tx.gasprice
+      expectedGasPrice: tx.gasprice,
+      adminFee: request.adminFee
```

Comment:
I'm not sure I completely understand the significance of storing the admin fee in the RequestBilling struct. Can you elaborate?

Reply:
It makes it so that the Coordinator doesn't need to call Router.getConfig(), the data is already available for it

Reply:
Great! I was thinking about that before actually and wondering if we should just have the updateConfig function in the FunctionsRouter call setAdminFee on the billing contract to save gas, but this is a better solution as I don't like the complexity of one contract updating config settings on another.